kf_lib_data_ingest/common/file_retriever.py

```diff
@@ -88,23 +90,59 @@ def _web_save(protocol, source_loc, dest_obj, auth=None, logger=None):
     """
     Get contents of a file from a web server to a local file-like object.
 
+    Preserves the name of the original file if Content-Disposition
+    header is provided in the response.
+    See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Disposition # noqa E501
+
     :param protocol: URL protocol identifier
     :type protocol: str
     :param source_loc: address or path
     :type source_loc: str
-    :param dest_obj: Receives the data downloaded from the source file
-    :type dest_obj: File-like object
+    :param dest_dir: Directory where temp file will be downloaded
+    :type dest_dir: str
+    :param delete: Flag indicating whether or not to delete file after download
+    :type delete: bool
```

Comment:
revert these param lines and the return line (though I'm not opposed to _also_ returning dest_obj. it just doesn't do that right now)

Reply:
None of the other getters return the dest obj so for now I'm going to revert back to what we had.